lib/datadog.py

```diff
@@ -305,10 +305,13 @@ def compile(install_path, cache_dir):
     download(install_path, cache_dir)
 
 
-def run():
+def run(runtime_version):
     if not is_enabled():
         return
 
+    if runtime_version < 7.14:
+        logger.error("Datadog integration requires Mendix 7.14 or newer")
```

Comment:
lets make this a warning. Append something like' Datadog agent is not activated.'